README.md

```diff
@@ -11,15 +11,12 @@ Read and run the setup script:
 ./bin/setup
 ```
 
-Load environment vars:
-```
-unset $(grep --no-filename --invert-match '^#' .env.shared .env | cut -f1 -d"=" | xargs)
-export $(grep --no-filename --invert-match '^#' .env.shared .env | xargs)
-```
```

Comment:
this actually doesn't work for quoted var values such as "foo bar", xargs removes the quotes.